tests/arch/arm/arm_runtime_nmi/src/arm_runtime_nmi.c

```diff
@@ -18,12 +18,13 @@
 
 extern void z_NmiHandlerSet(void (*pHandler)(void));
 
+int isr_cnt = -1;
+
 static void nmi_test_isr(void)
 {
 	printk("NMI received (test_handler_isr)! Rebooting...\n");
```

Comment:
Can you please remove the "Rebooting..." part as well. I don't see a reboot anywhere.